apps/src/code-studio/pd/workshop_dashboard/components/edit_enrollment_name_dialog.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Modal, Button, Row, Col, FormGroup} from 'react-bootstrap';
+import _ from 'lodash';
+
+export default class EditEnrollmentNameDialog extends React.Component {
+  static propTypes = {
+    selectedEnrollment: PropTypes.object,
+    show: PropTypes.bool,
+    onShow: PropTypes.func,
```

Comment:
It looks like the `onShow` prop isn't used.

Reply:
Oops, meant to delete. Thanks.